packages/selected-network-controller/tests/SelectedNetworkController.test.ts

```diff
@@ -742,11 +746,12 @@ describe('SelectedNetworkController', () => {
   });
 
   describe('Constructor checks for domains in permissions', () => {
-    it('should set networkClientId for domains not already in state', async () => {
+    it('should set networkClientId for domains not already in state when useRequestQueuePreference is true', async () => {
```

Comment:
Nit: Likewise here, we may want to test the opposite case

Reply:
addressed here https://github.com/MetaMask/core/pull/4388/commits/c30f5d86403c4cee572aaa9cea87f45b2f8a6b13